rfc/rfc014-authorization-credential.md

```diff
@@ -79,15 +79,17 @@ The `credentialSubject` field contains the following:
 
 ### 3.2.1 Required fields
 
-The `id`, `legalBase` and `purposeOfUse` fields MUST be filled. Within the `legalBase` object, `consentType` MUST either be `implied` or `explicit`. When `explicit`, the `evidence` and `subject` fields MUST be filled.
+The `id`, `legalBase` and `purposeOfUse` fields MUST be filled. Within the `legalBase` object, `consentType` MUST either be `implied` or `explicit`. When `explicit`, the `subject` field MUST be filled.
 
 ### 3.2.2 Scoping
 
 The `id` field MUST contain the DID of the actor. The `subject` field MAY contain the patient identifier. The example above uses the _oid_ for the Dutch citizenship number. The `purposeOfUse` field refers to an access policy. A Bolt MUST describe this policy as a set of FHIR resources that can be accessed with the credential. The `resources` array extends on the policy. It defines specific resources that may be accessed in addition to the policy. When no `subject` is given, the credential MUST contain `resources` that refer to individual resources. The contents of those individual resources MUST NOT contain any personal information. Without `subject` and `resources` it would be valid to request an access token with just the `purposeOfUse` in the JWT grant.
 
 ### 3.2.3 Legal base
 
-The patient consent can be either implied or explicit. When it's implied, this should be reflected by the `purposeOfUse`. A Bolt MUST therefore also describe if it is covered by explicit or implied consent. When the credential is given in the context of explicit consent, the `legalBase.evidence` MUST be filled. It MUST contain a value for the `path` and `type` fields. The `path` is a relative path to the service data endpoint and the `type` contains the media type as specified by [RFC6838](https://datatracker.ietf.org/doc/html/rfc6838). The evidence resource MUST be accessible with an access token that was created with the corresponding credential.
+The patient consent can be either implied or explicit. When it's implied, this should be reflected by the `purposeOfUse`. A Bolt MUST therefore also describe if it is covered by explicit or implied consent. 
+When `legalBase.evidence` is filled, it MUST contain a value for the `path` and `type` fields. The `path` is a relative path to the service data endpoint and the `type` contains the media type as specified by [RFC6838](https://datatracker.ietf.org/doc/html/rfc6838). 
```

Comment:
> The `path` is a relative path to the service data endpoint
This feels broken to. The assumption the evidence can be found on the same service feels wrong. What if the service endpoint changes? This should be either a full path or a relative path to a predefined endpoint name like "consent-evidence".

Reply:
That is broken, but there's no alternative yet